### ISSUE TITLE
Accessibility of reporting status page

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#989]
Accessibility issues from #970 

When zoomed in on reporting status page, some content overlaps:

![image](https://user-images.githubusercontent.com/43404358/100733682-71c1ce80-33c6-11eb-9692-ef311a88091a.png)

^^ This isn't an AA fail but it is recommended that we fix it

***

Structural mark-up may make the information easier to read. This does not have to affect the visual presentation at all.
Example:

```
<dl>
   <dt>
    <span class="status complete">18</span>
     <strong>Fully disaggregated</strong>
   </dt>
   <dd>
      <span class="value">7%</span>
   </dd>
   […]
</dl>
```

***

Another thing our screen reader users do often pick up on is splitting headings across multiple components, as this can be confusing and appear as separate headings.

![image](https://user-images.githubusercontent.com/43404358/100229996-ffac3e00-2f1c-11eb-80a2-84059a8cf62d.png)
 
Current Code Ref(s): #goalsview > div:nth-child(3) > div.details > h3

```
<h3 class="status-goal" data-element-id="headingsMap-3">
   <a href="/disagg-rs1/1">No poverty</a>
   <span class="total">14 indicators</span>
</h3>
```

I think this would work better as a `<hgroup>` as the text ‘No poverty’ and ‘14 indicators’ are really separate, with ‘14 indicators’ more of a subheading. 

Example:

```
<hgroup>
   <h3 class="status-goal" data-element-id="headingsMap-3">
      <a href="/disagg-rs1/1">No poverty</a>
   </h3>
   <h4><span class="total">14 indicators</span></h4>
</hgroup>
```

***

In Windows High Contrast mode, background colours invert to black, so some visual information is lost. You have text alternatives, but it may assist low vision users in identifying the purpose of some of this content if you made some small fixes. This includes the number of indicators lacking a border and I know in the initial audit, Low Vision users mistook the bar visualisation for a scrollbar.

![image](https://user-images.githubusercontent.com/43404358/100230124-33876380-2f1d-11eb-9a6d-73cb631b5b0d.png)

This is a quick fix as using a ‘border: 2px solid transparent’ will not affect the visual styling at all except for users in windows high contrast mode, as the borders become white. So it could look more like this:

![image](https://user-images.githubusercontent.com/43404358/100230153-3da96200-2f1d-11eb-9855-84fb4bae4700.png)

I think the borders around the indicators help, and the visualisation isn’t a perfect fix but at least it doesn’t look like a scrollbar!

***

The links for alternate colour schemes are indistinguishable in high Contrast Mode:

![image](https://user-images.githubusercontent.com/43404358/100230216-5580e600-2f1d-11eb-8268-6c012caf6b19.png)

I don’t think this is an issue as users would need to be making use of alternate colour schemes already to notice this, but I just thought I would mention it.

**Comment by brockfanning** [comment]
I've put up a PR for some of the changes requested here: #1044. However it doesn't include the `hgroup` change mentioned and doesn't include the `<dl>` change mentioned here.

Regarding the `hgroup` change - This seemed thorny because we also have an `<h2>` at the top with a span in it, and we would presumably need to do the same there. And if that turned into an `<h2>` and `<h3>`, then the others down the page would need to be `<h4>` and `<h5>` I assume. This seemed like a lot of headings, so instead I simply move the `<span>` with the indicator totals out of the headings.

Regarding the `dl` change - This markup change will require matching style changes to ensure that it continues to look the same. So we'd be changing style and markup, which has a real risk of being a breaking change if any countries are depending on the old markup or style. So this makes me uneasy. If it is necessary for AA compliance, then so be it. But if this is a AAA issue or just a suggestion - I think we should discuss the possibility of postponing it until version 2.0.0.

And a bit more on that: I'm not saying we can never do any markup/style changes until 2.0.0. It's just a judgement call we need to make for each proposed change. If the change has a significant risk of breaking sites, then it might not be a good idea. But if it is a minor enough change that there is not much risk, then it's probably OK. This is why I went ahead and moved some `<span>` elements out of headings. I thought that is probably minor enough that it's OK.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning closing as was complete in 1.3.0
